### PR TITLE
Updated with correct tags for image reference

### LINKED ISSUE #1783: Change the <a> tags to <image> tags
In the RankerMetrics the `<a>`  tags for images will need to get converted to `<image>` tags, so the images display. 

https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.Data/Evaluators/Metrics/RankerMetrics.cs#L23

### PR DESCRIPTION
**Pull request by dhilmathy**
Updated with correct tags for image. `<a>` to `<image>`

Fixes #1783

- [x] There's a descriptive title that will make sense to other developers some time from now. 
- [x] There's associated issues. All PR's should have issue(s) associated - unless a trivial self-evident change such as fixing a typo. You can use the format `Fixes #nnnn` in your description to cause GitHub to automatically close the issue(s) when your PR is merged.
- [x] Your change description explains what the change does, why you chose your approach, and anything else that reviewers should know.
- [ ] You have included any necessary tests in the same PR.

### CODE REVIEW

**Comment by Ivanidzo4ka** [comment]
Thank you @dhilmathy!

**Comment by Ivanidzo4ka** [comment]
@sfilipi, @Zruty0 , @abgoswam  can one of approve?

File: src/Microsoft.ML.Data/Evaluators/Metrics/RankerMetrics.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ public sealed class RankerMetrics
         /// <a href="https://en.wikipedia.org/wiki/Discounted_cumulative_gain">Discounted Cumulative gain</a>
         /// is the sum of the gains, for all the instances i, normalized by the natural logarithm of the instance + 1.
         /// Note that unline the Wikipedia article, ML.Net uses the natural logarithm.
-        /// <a href="https://github.com/dotnet/machinelearning/tree/master/docs/images/DCG.png"></a>
+        /// <image href="https://github.com/dotnet/machinelearning/tree/master/docs/images/DCG.png"></image>
```
Comment (Zruty0) [inline src/Microsoft.ML.Data/Evaluators/Metrics/RankerMetrics.cs]: is 'href' appropriate now? Or should it be 'src' ?

Comment (danmosemsft) [reply src/Microsoft.ML.Data/Evaluators/Metrics/RankerMetrics.cs]: It should be src.

Comment (dhilmathy) [reply src/Microsoft.ML.Data/Evaluators/Metrics/RankerMetrics.cs]: I'd updated with 'src'

**Pull request merged**